### PR TITLE
CASMTRIAGE-7296 new add documentation for the ceph-upgrade-tool.py

### PR DESCRIPTION
**Pull request by leliasen-hpe**
# Description

The ceph-upgrade-tool.py is used as part of the storage node upgrade. This PR removes documentation for the `cubs_tool.py` and adds documentation for the `ceph-upgrade-tool.py`.

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/utility_storage/Ceph_upgrade_tool_Usage.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# `ceph-upgrade-tool.py` Usage
+
+* [Introduction](#introduction)
+* [Usage](#usage)
+* [What the `ceph-upgrade-tool.py` does](#what-the-ceph-upgrade-toolpy-does)
```
Comment (rustydb) [inline operations/utility_storage/Ceph_upgrade_tool_Usage.md]: Easier to read, would obviously need to adjust the header to match
```suggestion
* [What is `ceph-upgrade-tool.py`](#what-is-ceph-upgrade-toolpy)
```

**Review by rustydb — CHANGES_REQUESTED** [review]

File: operations/utility_storage/Ceph_upgrade_tool_Usage.md (hunk 1 above)
Comment (leliasen-hpe) [reply operations/utility_storage/Ceph_upgrade_tool_Usage.md]: Updated

**Review by rustydb — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

**Comment by leliasen-hpe** [comment]
I manually backported this to release/1.4:https://github.com/Cray-HPE/docs-csm/pull/5412 and release/1.3: https://github.com/Cray-HPE/docs-csm/pull/5413.

**Review by dborman-hpe — APPROVED** [review]
LGTM

**Pull request merged**